src/main/java/eod/warObject/character/concrete/blue/HeavyPolice.java

```diff
@@ -27,6 +39,54 @@ public String getName() {
     @Override
     public void attack(EffectExecutor executor) {
         super.attack(executor);
-        // TODO
+        executor.tryToExecute(
+                RequestRegionalAttack(attack).from(this).to(getAttackRange())
+        );
+
+        afterAttack();
+    }
+
+    @Override
+    public ArrayList<Point> getAttackRange() {
+        PointParam param = new PointParam();
+        param.range = 1;
+        return player.getBoard().get4Ways(position, param);
+    }
+
+    public class DamageLessThan2 implements EventReceiver {
+        private HeavyPolice holder;
+        private ArrayList<Class<? extends Event>> canHandle;
+
+        public DamageLessThan2(HeavyPolice holder) {
```

Comment:
You don't have to pass HeavyPolice in.
Java can access outside class by `Outside.this`
In this case `HeavyPolice.this`

Reply:
fixed